include/cuco/static_multimap.cuh

```diff
@@ -728,9 +730,9 @@ class static_multimap {
      */
     __host__ __device__ device_view(pair_atomic_type* slots,
                                     std::size_t capacity,
-                                    Key empty_key_sentinel,
-                                    Value empty_value_sentinel) noexcept
-      : view_base_type{slots, capacity, empty_key_sentinel, empty_value_sentinel}
+                                    sentinel::empty_key<Key> empty_key_sentinel,
+                                    sentinel::empty_value<Value> empty_value_sentinel) noexcept
+      : view_base_type{slots, capacity, empty_key_sentinel.value, empty_value_sentinel.value}
```

Comment:
```suggestion
      : view_base_type{slots, capacity, empty_key_sentinel, empty_value_sentinel}
```

Reply:
I've recently updated the suggestions in a single commit. Unfortunately I've had to call `git pull --rebase` to sync with the current dev branch.

Reply:
I did not notice the "Add suggestion to batch" button :man_facepalming:, my bad...